components/nds-filters/__tests__/__snapshots__/nds-filters.test.js.snap

```diff
@@ -37,54 +37,55 @@ Array [
         </h2>
```

Comment:
Please can we also update the type defs for the new sorting options? e.g. https://github.com/nice-digital/nice-design-system/blob/main/components/nds-filters/nds-filters.d.ts#L13